src/ayb_db/db_interfaces.rs

```diff
@@ -247,6 +260,36 @@ WHERE id = $1
                 Ok(entity)
             }
 
+            async fn update_entity_by_id(&self, entity: &Entity, entity_id: i32) -> Result<InstantiatedEntity, AybError> {
+                let entity: InstantiatedEntity = sqlx::query_as(
+                    r#"
+UPDATE entity SET
+    description = $1,
+    workplace = $2,
+    display_name = $3,
+    links = $4
+WHERE entity.id = $5
+RETURNING id, slug, entity_type, display_name, description, workplace, links
+                    "#
+                )
+                .bind(&entity.description)
+                .bind(&entity.workplace)
+                .bind(&entity.display_name)
+                .bind(serde_json::to_value(&entity.links).unwrap())
```

Comment:
safe unwrap?

Reply:
> Safe unwrap?

Yes, this unwrap does not refer to the datatype, this unwrap can only panic if the struct being serialized contains a `HashMap` whose keys are not strings.
That's not the case, so it's safe to unwrap here :)
I don't know if AybError implements `From<serde_json::Error>`, if that's the case I'll replace the unwraps with the try operator (`?`)

I think the datatype is checked by sqlx

Reply:
Perfect! Thank you!